src/client/app/component/DisplayImage/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './style.scss';
+import * as imagesJson from './imagesForTest.json';
+
+const imagesList = imagesJson.imagesUrl;
+
+class DisplayImage extends React.Component {
```

Comment:
`DisplayImage` should be `ImageDisplay` or something similar. Classes and variables should be nouns and functions should be verbs